WordPressAuthenticator/Unified Auth/View Related/Google/GoogleSignupConfirmationViewController.swift

```diff
@@ -46,13 +46,62 @@ private extension GoogleSignupConfirmationViewController {
         submitButton?.accessibilityIdentifier = "Google Signup Email Next Button"
     }
     
-    // MARK: - Button Action Handling
+    // MARK: - Button Handling
 
     @IBAction func handleSubmit() {
-        displayError(message: "Cheers!")
-        // TODO: create account.
-        // TODO: re-enable this when handling added.
-        // configureSubmitButton(animating: true)
+        configureSubmitButton(animating: true)
+        GoogleAuthenticator.sharedInstance.delegate = self
+        GoogleAuthenticator.sharedInstance.createGoogleAccount(loginFields: loginFields)
+    }
+
+}
+
+// MARK: - GoogleAuthenticatorDelegate
+
+extension GoogleSignupConfirmationViewController: GoogleAuthenticatorDelegate {
+    
+    // MARK: - Signup
+    
+    func googleFinishedSignup(credentials: AuthenticatorCredentials, loginFields: LoginFields) {
+        self.loginFields = loginFields
+        showSignupEpilogue(for: credentials)
+    }
+
+    func googleLoggedInInstead(credentials: AuthenticatorCredentials, loginFields: LoginFields) {
+        self.loginFields = loginFields
+        showLoginEpilogue(for: credentials)
+    }
+    
+    func googleSignupFailed(error: Error, loginFields: LoginFields) {
+        configureSubmitButton(animating: false)
+        self.loginFields = loginFields
+
+        // Display generic inline error.
+        displayError(message: NSLocalizedString("Google sign up failed.", comment: "Message shown on screen after the Google sign up process failed."))
+        // Display the API error in a Fancy Alert.
+        displayError(error as NSError, sourceTag: .wpComSignup)
+    }
+    
+    // MARK: - Login
+
+    func googleFinishedLogin(credentials: AuthenticatorCredentials, loginFields: LoginFields) {
+        // Here for protocol compliance.
```

Comment:
Would it be applicable to break the protocol into two (Login and Signup perhaps), in order to avoid empty methods? Maybe it's not as I don't have the full context. Certainly can be done in a separate PR, if ever.

Reply:
Yea, like I said in my note:

> the way I implemented GoogleAuthenticatorDelegate didn't quite work out as planned, which resulted in several no-op delegate methods. I'll look at refactoring that separately.

I had initially thought `GoogleAuthenticatorLoginDelegate` and `GoogleAuthenticatorSignupDelegate` would only be needed for the original separate flows, but the "unified" `GoogleAuthenticatorDelegate` methods ended up being separated across VCs anyway. So I'll probably fallback to having just the 2 (login and signup). I'm not sure yet, which is why I'm going to to it separately.